src/otx/algo/action_classification/movinet.py

```diff
@@ -32,16 +38,41 @@ def __init__(
         metric: MetricCallable = MultiClassClsMetricCallable,
         torch_compile: bool = False,
     ) -> None:
-        config = read_mmconfig("movinet")
+        self.load_from = "https://github.com/Atze00/MoViNet-pytorch/blob/main/weights/modelA0_statedict_v3?raw=true"
```

Comment:
Is the reason for loading this pretrained weight from the personal repository that this model's official implementation is tensorflow, not pytorch?

Reply:
This might be because the movinet backbone implementation comes from there as described in https://github.com/openvinotoolkit/training_extensions/blob/1e1a9c5a6fbef7db03ca42af25b0fe4168ad0077/src/otx/algo/action_classification/backbones/movinet.py#L3. The original code also downloads from https://github.com/openvinotoolkit/training_extensions/blob/1e1a9c5a6fbef7db03ca42af25b0fe4168ad0077/src/otx/algo/action_classification/mmconfigs/movinet.yaml#L1.

Reply:
I'm just wondering if this way is fine.
If not, we can consider saving this weight into our storage server.
Let's discuss it later.

Reply:
alright, replacing pretrained weights is one of our next goal.